e2e/src/acceptanceTest/kotlin/maru/e2e/CliqueToPosTest.kt

```diff
@@ -81,24 +90,30 @@ class CliqueToPosTest {
     }
 
     private fun deleteGenesisFiles() {
+      val maruGenesis = File(genesisDir, "genesis-maru.json")
       val besuGenesis = File(genesisDir, "genesis-besu.json")
       val gethGenesis = File(genesisDir, "genesis-geth.json")
       val nethermindGenesis = File(genesisDir, "genesis-nethermind.json")
+      maruGenesis.delete()
       besuGenesis.delete()
       gethGenesis.delete()
       nethermindGenesis.delete()
     }
 
+    private fun containerShortNameToFullId(containerShortName: String) = containerShortName
+
     @BeforeAll
     @JvmStatic
     fun beforeAll() {
+      log.info("Using Maru as docker container: $useMaruContainer")
       deleteGenesisFiles()
       dataDir.deleteRecursively()
+      dataDir.mkdirs()
       qbftCluster.before()
-
       pragueSwitchTimestamp = parsePragueSwitchTimestamp()
-      dataDir.mkdirs()
-      maru = MaruFactory.buildTestMaru(pragueSwitchTimestamp)
+      if (!useMaruContainer) {
```

Comment:
Minor: a slightly cleaner way to do it would be to define a parent abstract class with the tests themselves and have 2 separate classes. Only one of which would be active based on `useMaruContainer`

It's a quite complex refactoring though and I don't think it provides enough value ATM, so I'm fine with the way things are